src/main/java/com/google/sps/servlets/utils/CommonWordsRetriever.java

```diff
@@ -14,18 +14,31 @@
 
 package com.google.sps.servlets.utils;
 
+import java.io.File;
 import java.util.ArrayList;
-import java.util.Arrays;
 import java.util.List;
+import java.util.Scanner;
 
-/** Class provides commonly used words that should be ignored within the word map */
+/** Class that provides commonly used words that should be ignored within the word map */
 public class CommonWordsRetriever {
```

Comment:
is this a static class as well?

Reply:
Since it's not a nested class it can't be declared as static